### PR TITLE
Update expectation for confirmation emails on multisite - MAILPOET-4667

### PR DESCRIPTION
**Pull request by triple0t**
## Description

Read more on [slack](https://a8c.slack.com/archives/C01GH835U90/p1666943880983079)

## Code review notes

 Mostly acceptance test fixes

## QA notes

 We may merge directly, as this is only to fix test failure.

## Linked PRs

https://github.com/mailpoet/mailpoet/pull/4436

### CODE REVIEW

File: mailpoet/tests/acceptance/Settings/WooCommerceSetupPageCest.php [hunk 1]
```diff
@@ -92,6 +92,8 @@ public function noCustomersBehaviourTest(\AcceptanceTester $i) {
 
   public function setupPageFormBehaviourTest(\AcceptanceTester $i) {
     $order = $this->orderFactory->create();
+    // make sure we have a registered customer for import toggle to show up
+    $customer = $this->customerFactory->create();
```
Comment (samnajian) [inline mailpoet/tests/acceptance/Settings/WooCommerceSetupPageCest.php]: Redundant variables `$order` && `#customer` can be removed

**Review by samnajian — DISMISSED** [review]
LGTM other than the recommended change.

**Comment by samnajian** [comment]
@triple0t This can be directly merged to trunk since it's only updating tests, and the comment is only a recommendation, so it's up to you!

**Review by samnajian — APPROVED** [review]
LGTM!

**Comment by triple0t** [comment]
Thank you for the review, @samnajian 

> This can be directly merged to trunk since it's only updating tests, and the comment is only a recommendation, so it's up to you!

Not a problem. I have pushed a new commit for that.

Thanks

**Pull request merged**